lib/modules/datasource/custom/formats/html.ts

```diff
@@ -0,0 +1,45 @@
+import is from '@sindresorhus/is';
+import { readLocalFile } from '../../../../util/fs';
+import { parse } from '../../../../util/html';
+import type { Http } from '../../../../util/http';
+import type { ReleaseResult } from '../../types';
+import type { CustomDatasourceFetcher } from './types';
+
+function extractLinks(content: string): ReleaseResult {
```

Comment:
@secustor should we move it to the class? 🤔

Reply:
Does not matter really IMO.